lib/net/ssh/connection/session.rb

```diff
@@ -583,9 +591,10 @@ def channel_close(packet)
         info { "channel_close: #{packet[:local_id]}" }
 
         channel = channels[packet[:local_id]]
+        channel[:remote_closed] = true
```

Comment:
I'm a bit bothered that we're using ```:remote_closed``` as a property here vs closed which is an attribute. Maybe ```local_closed``` and ```remote_closed``` should be normal attributes and we should just deprecate old ```closing?``` What do you think?